tools/lsp-wake/astree.cpp

```diff
@@ -446,40 +446,40 @@ SymbolKind ASTree::getSymbolKind(const char *name, const std::string &type) {
   }
 }
 
-void ASTree::mergeSameLineDefinitions(const SymbolDefinition &merge_from, SymbolDefinition &into) {
+void ASTree::mergeSameLineDefinitions(const SymbolDefinition &merge_from, SymbolDefinition *into) {
```

Comment:
What's the advantage to using a pointer? Is this just to match google style for our params?

This is a point where I disagree with the google style guide a bit because references have the advantage that they can't be null. So anything I always expect to be non-null I make a reference as quickly as possible. I don't care *that* much though so if you don't find that argument convincing then go ahead and land this.